BHoM/Attributes/InputAttribute.cs

```diff
@@ -40,38 +41,61 @@ public class InputAttribute : Attribute, IImmutable
 
         public virtual InputClassificationAttribute Classification { get; } = null;
 
+        public virtual UIExposure Exposure { get; private set; } = UIExposure.Display; //Default to allow all input attributes to be exposed by default
+
         /***************************************************/
         /**** Constructors                              ****/
         /***************************************************/
 
-        public InputAttribute(string name, string description, Type classification = null)
+        public InputAttribute(string name, string description)
+        {
+            Name = name;
+            Description = description;
+        }
+
+        /***************************************************/
+
+        public InputAttribute(string name, string description, UIExposure exposure)
+        {
+            Name = name;
+            Description = description;
+            Exposure = exposure;
+        }
+
+        /***************************************************/
+
+        public InputAttribute(string name, string description, UIExposure exposure = UIExposure.Display, Type classification = null)
```

Comment:
As discussed offline, I believe it is cleaner/safer to avoid default values in case of multiple constructors that differ between each other exactly by these parameters. Default values serve no purpose in this case, while they remove clarity from the code.

```suggestion
        public InputAttribute(string name, string description, UIExposure exposure, Type classification)
```